cart/infrastructure/defaultCartBehaviour.go

```diff
@@ -827,61 +892,22 @@ func (cob *DefaultCartBehaviour) collectTotals(cart *domaincart.Cart) {
 	}
 }
 
-// ApplyVoucher checks the voucher and adds the voucher to the supplied cart if valid
-func (DefaultVoucherHandler) ApplyVoucher(_ context.Context, cart *domaincart.Cart, couponCode string) (*domaincart.Cart, error) {
```

Comment:
should we move this to some sort of FakeVoucherHandler guess some of the integration tests are relying on this behaviour, same for giftcards